enterprise/app/org/create_org.tsx

```diff
@@ -19,8 +19,8 @@ export default class CreateOrgComponent extends OrgForm<grp.CreateGroupRequest>
     return new grp.CreateGroupRequest({ ...DEFAULT_VALUES, ...values });
   }
   async submitRequest() {
-    const { id } = await rpcService.service.createGroup(this.state.request);
-    await authService.setSelectedGroupId(id);
+    const rsp = await rpcService.service.createGroup(this.state.request);
+    await authService.setSelectedGroupId(rsp.id, rsp.url);
     router.navigateTo(Path.settingsPath);
```

Comment:
uber nit: this may be a losing battle since we are all used to Go, but for the JS code it's a bit unconventional to abbreviate stuff :P

```suggestion
    const response = await rpcService.service.createGroup(this.state.request);
    await authService.setSelectedGroupId(response.id, response.url);
    router.navigateTo(Path.settingsPath);
```